account_move_line_product/views/account_move_line_view.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```

Comment:
the &lt;data&gt; tag should be removed and the &lt;openerp&gt; tag renamed to &lt;odoo&gt; in version 9.